src/backend/distributed/commands/table.c

```diff
@@ -1063,7 +1129,8 @@ PreprocessAlterTableStmt(Node *node, const char *alterTableCommand,
 				 */
 				constraint->skip_validation = true;
 			}
-			else if (constraint->contype == CONSTR_PRIMARY)
+			else if (constraint->contype == CONSTR_PRIMARY || constraint->contype ==
```

Comment:
You are using this condition a lot
``` C
constraint->contype == CONSTR_PRIMARY ||
constraint->contype == CONSTR_UNIQUE ||
constraint->contype == CONSTR_EXCLUSION
```
It's better tp wrap it in a boolean function, something like the following:
https://github.com/citusdata/citus/blob/main/src/backend/distributed/deparser/deparse_function_stmts.c#L830

Reply:
Done.